### PR TITLE
Check file extension .json

### PR DESCRIPTION
**Pull request by tannerjt**
I kept getting error because my zipped up shapefile ended in the string json.  I made a small change to check for file extension not just 'json'.  Small fix, but now working without having to change my shapefile names.  Thanks for this great utility.

### CODE REVIEW

**Comment by calvinmetcalf** [comment]
i was doing that to catch files ending in .geojson so we should probably switch it to ===.json || .geojson

**Comment by tannerjt** [comment]
Ah, got it.  I updated the conditional to check for 'json' for a file extension, be it .geojson or .json.

File: lib/index.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ shp.parseZip = function(buffer, whiteList) {
 	}
 	var geojson = names.map(function(name) {
 		var parsed;
-		if (name.slice(-4).toLowerCase() === 'json') {
+		if (name.lastIndexOf('.') > -1 && name.substr(name.lastIndexOf('.')).indexOf('json') > -1) {
```
Comment (calvinmetcalf) [inline lib/index.js]: lets use slice instead of substr

**Comment by tannerjt** [comment]
Changed from .substr to .slice.  I agree, this follows more closely to the string methods you are using for other checking.

File: lib/index.js [hunk 2]
```diff
@@ -64,7 +64,7 @@ shp.parseZip = function(buffer, whiteList) {
 	}
 	var geojson = names.map(function(name) {
 		var parsed;
-		if (name.slice(-4).toLowerCase() === 'json') {
+		if (name.lastIndexOf('.') > -1 && name.slice(name.lastIndexOf('.')).indexOf('json') > -1) {
```
Comment (calvinmetcalf) [inline lib/index.js]: we can probably cache `name.lastIndexOf('.')`

**Comment by tannerjt** [comment]
Cached to local variable lastDotIdx for indexOf('.').

**Comment by calvinmetcalf** [comment]
looks good, will publish latter today

**Pull request merged**